Modules/blake2module.c

```diff
@@ -595,7 +616,7 @@ py_blake2b_or_s_new(PyTypeObject *type, PyObject *data, int digest_size,
 
     return (PyObject *)self;
 error:
-	Py_XDECREF(self);
+    Py_XDECREF(self);
```

Comment:
Stray change

Reply:
Yes but it's actually a tab converted into spaces (this would render differently on IDEs with different tab sizes)